src/main/resources/static/js/cart.js

```diff
@@ -8,13 +8,15 @@ const addCartItem = (productId) => {
 
     // TODO: [2단계] 장바구니 CRUD API에 맞게 변경
     axios.request({
-        url: '',
+        url: 'http://localhost:8080/carts/' + itemId,
```

Comment:
1. 불필요한 TODO는 모두 지워볼까요?
2. 상단에 아래와 같이 선언해두면 application에서 port를 변경하는 등의 작업을 해도, 해당 프로그램이 원활하게 동작할 수 있겠죠?

    ``` javascript
    const currentUrl = window.location.href;
    const url = new URL(currentUrl);
    const protocol = url.protocol;
    const hostname = url.hostname;
    const port = url.port;
    const currentServerUrl = protocol + "//" + hostname + ":" + port
    ```
    
    ```suggestion
            url: currentServerUrl + '/carts/' + itemId,
    ```
3. axios에서는 `axios.request()` 뿐만 아니라 `axios.get()`, `axios.post()` 등의 HTTP Method를 이용한 함수도 지원하는데요. `method: 'POST',`로 선언하지 말고 이를 이용해봐도 좋겠네요!

Reply:
A1. js를 처음 경험해봐서 어떤 부분을 수정했는지 기억해놓으려고 남겨뒀었는데.. ㅎㅎ 미션을 제출했으니 삭제하겠습니다 !

A2. 반영하겠습니다 !

A3. 1단계 미션에서는 `axios.get()`, `axios.post()`를 사용했었고 이번 미션에서는 request()로 한 번 해보고 싶었습니다 😀
두 방법 다 시도해본만큼 하나의 방법으로 통일하도록 하겠습니다 !!

Reply:
1. 👍 
2. 👍 URL을 명시하는 것과 PATH를 명시하는 것의 차이는 무엇이 있을까요? 
3. 이전 미션에서 get/post를 사용하신 줄 몰랐네요 😃 
이 부분은 개인 취향대로 가면 좋을 것 같네요! 👍

Reply:
A2. PATH는 리소스의 경로를 나타냅니다. 그래서 axios.get('/members/1') 라는 요청은 baseURL에 설정된 호스트를 기준으로 동작하고, URL을 명시하는 것은 명시된 URL을 바탕으로 동작하는 것 같습니다 !

Reply:
👍 

URL 직접 선언했을 때의 장점이, PATH를 명시했을 때보다 크다면 그대로 유지하면 될 것 같고,
따로 없다면 PATH로 선언하는 게 코드가 조금 더 간결해지겠죠?

+ `cart.html`에 하드코딩되어 있는 부분은 위에서 결정한 방식으로 통일시켜주면 좋을 것 같아요!

Reply:
반영하겠습니다 ㅎㅎ